pkg/compose/build_buildkit.go

```diff
@@ -71,3 +76,214 @@ func (s *composeService) doBuildBuildkit(ctx context.Context, project *types.Pro
 
 	return imagesBuilt, err
 }
+
+//
+// All code below is from https://github.com/docker/buildx/blob/v0.8.1/commands/util.go
+//
+
+// getDefaultDrivers returns drivers based on current cli config
+func getDefaultDrivers(ctx context.Context, dockerCli command.Cli, defaultOnly bool, contextPathHash string) ([]build.DriverInfo, error) {
```

Comment:
@tonistiigi would it make sense to export this func so we don't have to copy/paste this code?

Reply:
vendoring is indeed actually a form of copy/paste, but at least is automated :P
About invoking `buildx bake` from compose, this assumes the latter will parse the compose file the same way we do with in-memory changes, that could be an option... but not something we considered so far.
Also, Compose V2 can be used standalone, and then should be able to support `build` command without external dependency (but this is something we could reconsider)

Is there a way to invoke `buildx bake` passing directly solver options?

Reply:
>  same way we do with in-memory changes

I'm not sure what this means. We can probably start to track this in a separate issue.

> Is there a way to invoke buildx bake passing directly solver options?

The solver options come from parsing the compose files. Only the path may be needed to be passed if it is not default.

Reply:
There's not just a compose file: user can set overrides, configure variables for interpolations, and pass flags to amend the model in many ways, this is what I mean by "in memory model" - which we could marshall back to yaml, but this assumes bake is up to date with the compose-spec, which we can't guarantee.